applications/adv_networking_bench/cpp/default_bench_op_rx.h

```diff
@@ -184,24 +184,14 @@ class AdvNetworkingBenchDefaultRxOp : public Operator {
 
     auto burst_size = adv_net_get_num_pkts(burst);
 
-    // If packets are coming in from our non-GPUDirect queue, free them and move on
-    // hardcoded to match the YAML config files in this sample app.
-    // NOTE: we can't actually ignore all standard linux packets on a real network (with a switch),
-    //       at least ARP packets should be processed, or delegate to linux for standard traffic.
-    if (adv_net_get_q_id(burst) == 0) {
-      adv_net_free_all_pkts_and_burst(burst);
-      return;
-    }
-
-    // Count packets received
-    ttl_pkts_recv_ += burst_size;
-
```

Comment:
I don't think we should remove this without removing the queue 0 from other config files. Maybe out of scope of this MR? Would like to at least separate it in different commits so that anyone can come back to history and understand why that was done

Reply:
This should be removed and we should add the flow_isolation parameter into each of them so it's not confusing to users to have that code.

Reply:
agreed. or default `flow_isolation` to true. I believe we have jira tickets for all this, are you ok doing this in a separate MR?